### DISCUSSION TITLE
操作系统模型与乐高积木 · OSDI 2018 - 面向信仰编程 · /papers-legoos

### DISCUSSION THREAD

**Discussion by draveness** [draveness/blog-comments#622] [Announcements]
https://draveness.me/papers-legoos/

**Comment by shanzhiliu** [comment]
大佬，想问下你这个博客使用是自己开发的吗，还是基于什么框架写的。

**Comment by draveness** [comment]
> 大佬，想问下你这个博客使用是自己开发的吗，还是基于什么框架写的。

hugo

**Comment by Gnosnay** [comment]
这个主题好好看！刚想问文中配图是怎么做的，就看到了文末的ref
每次看到博主的文章，都收获慢慢，感谢！

**Comment by flyfy1** [comment]
> @Yansongsongsong
> 这个主题好好看！刚想问文中配图是怎么做的，就看到了文末的ref
> 每次看到博主的文章，都收获慢慢，感谢！

感觉博主看到这种问题会哭笑不得了……每一篇文章末尾都有这个问题哈哈

之前博主写过一篇博文，还专门写过一篇关于文章配图的文章 - https://draveness.me/sketch-and-sketch/

**Comment by Gnosnay** [comment]
嗯 是的 最后看到了ref
博主的文章真的给了很多帮助

**Comment by anotherNight** [comment]
好文，后面的索引材料也是干货！

**Comment by furthergo** [comment]
赞👍，关注博主很久了，想请教博主两个问题：
1. 写这样一篇文章大概要多久的时间？我最近也在写，发现在提前准备过的情况下，写一篇比较简陋的都要至少六七个小时，像这样一篇需要画图、找reference、写文章感觉要花很大的精力，过程中会有什么额外的收获吗？
2. 我现在是做iOS客户端的，最近准备转服务端尝试一下。我看博主之前也是做iOS客户端的，请问一下这种方向的转变有哪些可以分享一下的经验或者说是建议吗？

非常感谢！

**Comment by draveness** [comment]
> 赞👍，关注博主很久了，想请教博主两个问题：
> 
> 1. 写这样一篇文章大概要多久的时间？我最近也在写，发现在提前准备过的情况下，写一篇比较简陋的都要至少六七个小时，像这样一篇需要画图、找reference、写文章感觉要花很大的精力，过程中会有什么额外的收获吗？

不一定，看文章长度吧，写的时候可以让自己的思路更清晰

> 2. 我现在是做iOS客户端的，最近准备转服务端尝试一下。我看博主之前也是做iOS客户端的，请问一下这种方向的转变有哪些可以分享一下的经验或者说是建议吗？
> 
> 非常感谢！

多看多学，第一步可能稍微比较难，后面其实都很顺利

**Comment by yeya24** [comment]
图2的标题有个typo,不是logoOS

---

*2020-09-21 UPDATES：已修复*
